server/game/cards/locations/04/renlyspavilion.js

```diff
@@ -0,0 +1,54 @@
+const DrawCard = require('../../../drawcard.js');
+
+class RenlysPavilion extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Kneel this card',
+            method: 'kneel'
+        });
+    }
+
+    kneel(player) {
+        if(this.location !== 'play area' || this.kneeled) {
+            return false;
+        }
+        this.game.promptForSelect(player, {
+            cardCondition: card => this.cardCondition(card),
+            activePromptTitle: 'Select a character to get -1 STR',
+            waitingPromptTitle: 'Waiting for opponent to use ' + this.name,
+            onSelect: (player, card) => this.lowerStr(player, card)
+        });
+        this.game.promptForSelect(player, {
```

Comment:
Our general style has been to have these selections happen sequentially (i.e. onSelect of one starts the second prompt) rather than in parallel. Unfortunately that'll make it even more verbose.